components/Team.tsx

```diff
@@ -132,6 +169,48 @@ const Team = ({ team, profile }: Props) => {
           {!!error && <ErrorMessage>{error}</ErrorMessage>}
         </Column>
       </Flex>
+      <Flex direction="row" tabletVertical justify="space-between">
+        <Column flexBasis={48}>
+          <Header>Edit Team Visibility</Header>
+          <ChangeProfileOption>
+            <input
+              name="visibility"
+              type="checkbox"
+              checked={visible}
+              onChange={handleChangeVisibility}
+            />
+            <ShareProfileText>
+              Share the team name and description with hackers looking for teams!
```

Comment:
Can we change this to `Share the team name and description with hackers looking to join a team`?